fix: add multiple blackbox tests and fix small bugs (#371)
asyncapi/modelina: docs/development.md

- `npm run docker:build` builds the docker image with the tag `asyncapi/modelina` (the rest of the scripts run this one as well).
- `npm run docker:test` runs the main test suite.
- `npm run docker:test:blackbox` runs the BlackBox test suite.

## Environment setup

To setup the environment follow these steps:
1. Setup the project by first installing the dependencies `npm install`
2. Make sure the tests pass by running `npm run test` script
    - You can update snapshots by running `npm run test -- -u`
3. Make sure code is well formatted and secure `npm run lint`

## BlackBox testing

> We have a bunch of BlackBox tests, that are run separately to the `npm run test` script. Please refer to the [BlackBox documentation](../test/blackbox) for further information.

Reviewer: ```suggestion
We have several BlackBox tests that are run separately from the `npm run test` script. Please refer to the [BlackBox documentation](../test/blackbox) for further information.
```
